plugins/auto-sizes/tests/test-optimization-detective.php

```diff
@@ -3,6 +3,8 @@
  * Tests for auto-sizes plugin's optimization-detective.php.
  *
  * @package auto-sizes
+ *
+ * @noinspection PhpUnhandledExceptionInspection
```

Comment:
That's a PHPStorm thing, right? Do we usually have IDE-specific annotations?

Reply:
Yeah, elsewhere in the codebase I've been using them.